Esri/arcgis-runtime-samples-ios
# [Fix] Add missing ODR tag and remove unused empty tags

## Description

This PR fixes various issues related to ODR tags.

- It removes 4 unused tags: Bristol, MobileMapPackages, PublicArtShapefile, TilePackages 
    - The public art tag should have been removed as part of https://github.com/Esri/arcgis-runtime-samples-ios/pull/1156 , but wasn't
    - All samples using `Bristol.dae` Esri model plane are actually using the `Bristol3DModel` resource tag, so it is safe to remove `Bristol`. See https://github.com/Esri/arcgis-runtime-samples-ios/pull/263
    - `MobileMapPackages` was introduced as part of https://github.com/Esri/arcgis-runtime-samples-ios/pull/255 , but it seems later the resource tag got renamed to `YellowstoneMMPK`, which made the old tag unused
    - Same for `TilePackages` that was also introduced in #255 but not removed
- It adds a missing tag which resource is used by `Navigate a route with rerouting` sample, missing in original PR https://github.com/Esri/arcgis-runtime-samples-ios/pull/988 [here](https://github.com/Esri/arcgis-runtime-samples-ios/pull/988/files#diff-d392130cdb72c7610a87b8808fd2a32fa7ea4af78b97e288697d78e500160dfbR1371)

## Linked Issue(s)

- `common-samples/issues/3734`, thanks to @Gela 's findings!

An empty tag looks like this in the project file. I verified and removed all empty ones in this PR.

![iShot_2022-08-01_16 30 30](https://user-images.githubusercontent.com/9660181/182263415-607de760-9055-44b6-af36-1f86f1a79678.png)

## How To Test

- I'll add Phil as another pair of eyes to double check what I've done here is correct.
- The rerouting sample should run on Simulator, and will run when we upload a TestFlight/release to App Store.  Currently the App Store version gives invalid call error, due to a missing on-demand San Diego network data (sandiego.tn). We should conduct the test again in a few weeks along with 100.15 release @vquach2404 
- No other changes should happen to any sample
